prisma/schema.prisma

```diff
@@ -90,6 +90,40 @@ model UserSession {
   expiresAt DateTime
 }
 
+// WebhookWorker can be used to handle workflows via webhooks
+model WebhookWorker {
+  id        String   @id @unique @default(uuid()) @db.Uuid
+  createdAt DateTime @default(now())
+  updatedAt DateTime @default(now()) @updatedAt
+
+  name       String
+  secret     String
+  url        String  @unique
+  // stores the encrypted hatchet auth token
+  tokenValue String?
```

Comment:
why are we storing this here and also in token?

Reply:
Where in `token` do you mean?

Reply:
my fault